### DISCUSSION TITLE
Links to Microsoft Learn

### DISCUSSION THREAD

**Discussion by paulushub** [dotnet/docfx#9284] [Q&A]
**Describe the bug**
Links to Microsoft Learn or MSDN classes generating warning. I have used the following links in my previous DocFX project and all resolved correctly.
What changed in the current the support of the links?
```
[XmlnsPrefixAttribute](xref:System.Windows.Markup.XmlnsPrefixAttribute) and 
[XmlnsDefinitionAttribute](xref:System.Windows.Markup.XmlnsDefinitionAttribute)
```
The result is here: https://elinamllc.github.io/SharpVectors/articles/getting_started.html#xaml-output---xmlxamlwriter

**To Reproduce**
1. docfx init -q
2. add the links shown above to any article
3. docfx build docfx_project\docfx.json --serve

**Expected behavior**
A resolved links to XmlnsPrefixAttribute and XmlnsDefinitionAttribute MSDN classes.

**Context (please complete the following information):**
- OS: Windows
- Docfx version: v2.71.0

**Additional context**
![image](https://github.com/dotnet/docfx/assets/809351/3c7a6358-42a2-49aa-b5fa-ffc0721dd701)

**Comment by yufeih** [comment]
Is it related to https://github.com/dotnet/docfx/issues/8958?

**Reply by paulushub** [reply]
Possibly, Thanks for the reply.
However, I have tried some of the xrefService services, but seems I am missing something.

With the current changes, what is the recommended way to use the xref links to the MS-Learn?

**Reply by paulushub** [reply]
@yufeih This issue is not related to the https://github.com/dotnet/docfx/issues/8958.
Currently, links to every Microsoft Learn Documents is resolved without a problem, except those two attributes.
Please can you investigate this issue? 
I have hardcoded the links for now, having played many times with the filters without a success.
